.github/workflows/lint.yml

```diff
@@ -0,0 +1,14 @@
+name: lint
+
+on: push
+
+jobs:
+  lint:
+    name: Lint k8s-charts
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
can you leave a comment on what does this line do?

Reply:
I don't think it's useful. This is the very basic of github actions. 
If you don't know what this is doing, you don't know github actions at all and you have to learn github actions. It's like asking for comment on what does `class` keyword means in Java.

Reply:
I think it's useful. In general, we write a comment when we clone a repo in our bash scripts. The analogy for the `class` in Java is not correct. Even when we write class files in java, we have a documentation section for those classes describing what they are supposed to do. So, a similar idea should be valued in this line. You don't need to explain what the `uses` keyword is for. but mentioning that this step we are cloning the Github repository.

Reply:
https://github.com/commercetools/k8s-charts/pull/43/commits/93746aa15c308f6869d23c93ea1ca8d17a7c0e77